scripts/nfs-roles-management/internal/synchronization/synchronization.py

```diff
@@ -122,6 +127,10 @@ def synchronize_user(self, user, use_symlinks=False):
                     if storage_user.username.strip().lower() == user.strip().lower():
                         syncing_storages[storage] = storage_user_mask
 
+            if not details or details.blocked:
```

Comment:
just minor comment on naming, as for me `user_details` would be easier to follow here.